src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -152,22 +152,22 @@ public static void LogSectionsWithPaths(Dictionary<string, VirtualSettingSection
                 {
                     logger.LogMinimal($" file: {configPathsGroup.Key}");
                     LogSectionItems(configPathsGroup, logger);
-                    logger.LogMinimal("\n");
+                    logger.LogMinimal(Environment.NewLine);
                 }
             }
         }
 
         /// <summary>
         /// Logs each section of the configuration settings that will be applied.
         /// </summary>
-        public static void LogSectionsNoPaths(Dictionary<string, VirtualSettingSection> sections, ILogger logger)
+        public static void LogSectionsNoPaths(IEnumerable<string> sections, Settings settings, ILogger logger)
         {
             foreach (var section in sections)
             {
-                logger.LogMinimal(section.Key + ":");
-                var items = section.Value.Items;
+                logger.LogMinimal(section + ":");
+                var items = settings.GetSection(section).Items;
```

Comment:
Guard against null here too.

Reply:
I think it's fine? There aren't currently any codepaths that I'm noticing which will pass a null.